src/modules/navigator/mission_feasibility_checker.cpp

```diff
@@ -161,7 +161,9 @@ bool MissionFeasibilityChecker::checkGeofence(dm_item_t dm_current, size_t nMiss
 				return false;
 			}
 
-			if (!geofence.inside_polygon(missionitem.lat, missionitem.lon, missionitem.altitude)) {
+			if (missionitem.nav_cmd < NAV_CMD_DO_JUMP &&
```

Comment:
We could use https://github.com/PX4/Firmware/blob/master/src/modules/navigator/mission_block.cpp#L318 and also add NAV_CMD_DO_JUMP to the list there (it's currently not because it's handled separately in missions but adding it won't have any other effect)